src/index.ts

```diff
@@ -112,7 +113,7 @@ function run(updateDetails?: IUpdateDetails) {
 
     const mainRepoPath = AssetsCompiler.getMainRepoPath(process.cwd(), cli.flags.localonly);
     if (mainRepoPath === null) {
-        console.error(cerr`Failed to find path to main repository with cf.cplace.platform plugin...`);
+        console.error(cerr`Failed to find path to main repository containing cf.cplace.platform plugin... (Did you cd into the root directory of your cplace project?)`);
```

Comment:
This formulation sounds like cd'ing into the root is something to be avoided, when that is actually what needs to be done.
```suggestion
        console.error(cerr`Failed to find path to main repository containing cf.cplace.platform plugin... Ensure that you are starting from the root directory of your cplace project.`);
```